.env.example

```diff
@@ -0,0 +1,2 @@
+ACME_CA_SERVER="https://acme.proeftuin.uzi-online.rdobeheer.nl/"
+YUBIKEY_PIN="123456"
```

Comment:
Would be nice if we also define the proeftuin environment here as a base url.

https://github.com/minvws/nl-uzipoc-yubisign/blob/941458f0196a345381426524e95a2a7e6cc37e5e/app/page/logindigid.py#L36

`https://proeftuin.uzi-online.rdobeheer.nl`

Reply:
Yes, but we can't use the new ACME server for that.  So the only difference here would be an extra variable.

Reply:
Instead of the new added variable  `_PROEFTUIN_OIDC_LOGIN_URL = "https://proeftuin.uzi-online.rdobeheer.nl/oidc/login"`, it would be nice to define a base url in the environment file `https://proeftuin.uzi-online.rdobeheer.nl` so that if the url changes it is easy to configure it.

That base url can then be used for the following values:
https://github.com/minvws/nl-uzipoc-yubisign/blob/941458f0196a345381426524e95a2a7e6cc37e5e/app/page/logindigid.py#L58
https://github.com/minvws/nl-uzipoc-yubisign/blob/941458f0196a345381426524e95a2a7e6cc37e5e/app/page/logindigid.py#L61
https://github.com/minvws/nl-uzipoc-yubisign/blob/941458f0196a345381426524e95a2a7e6cc37e5e/app/page/logindigid.py#L83